lib/core/src/lib/comments/comment-list/comment-list.component.scss

```diff
@@ -1,68 +1,70 @@
-.adf {
-    &-comment-img-container {
-        width: 40px;
-        height: 100%;
-        display: flex;
-        align-self: flex-start;
-        padding-top: 18px;
-    }
+.adf-comment-list {
+    .adf {
+        &-comment-img-container {
+            width: 40px;
+            height: 100%;
+            display: flex;
+            align-self: flex-start;
+            padding-top: 18px;
+        }
 
-    &-comment-list-item {
-        white-space: initial;
-        display: table-row-group;
-        padding-top: 12px;
-        overflow: hidden;
-        height: 100% !important;
-        background-position: center;
-    }
+        &-comment-list-item.mat-list-item {
```

Comment:
sorry, I did not notice this one earlier, can this mat class be avoided?

Reply:
fixed